www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Next generation web applications with async PHP
+info: https://phpconference.com/php-core-coding/next-generation-web-applications-with-async-php/
+location: Berlin
+tags:
+  - next generation web applications with async php 
+  - reactphp
+  - frameworkx
```

Comment:
Okay as-is, I suppose?

Reply:
The type of event has always been tagged so far. Accordingly, I would suggest to keep #conference.